packages/health-check-runner/index.ts

```diff
@@ -0,0 +1,61 @@
+const exec = require("child_process").exec;
+import { Logger } from "@uma/financial-templates-lib";
+
+let logger;
+
+export async function run(): Promise<void> {
```

Comment:
Just OOC, why couldn't we run all of the inner health check commands directly from the Hub/Spoke rather than batching them in here? As far as I can see, they're just generic node commands, and the Spoke already knows how to call those, right?

Reply:
this is already going to run in the hub and spoke but it lets us define _one_ serverless job to run a set of "health checks", rather than needing to have each check be it's own spoke job. the main benefits are in the logging: we don't need to worry about how the health checks log: they just need to produce: a) exit codes relating to their output and b) console logs (standard out). this runner then executes them and wraps the logic to group logs together and process the exit codes.

Reply:
I totally hear you. Only pushing on reducing layers/complexity, since this code feels pretty similar to our spoke code. I think one of the issues we've had in the past is that fanciness around log processing and layers of subprocesses has resulted in a) fewer people understanding the architecture/what's happening and b) more bugs/maintainability issues in the code.

I don't have a strong opinion here, and I can see the argument for doing it both ways, I just want to make sure we're thinking about the layering complexity and considering alternatives!